### ISSUE TITLE
Include "How To Work With Ushahidi" on Plans page

### ISSUE THREAD

**Issue by caharding** [ushahidi/platform#2625]
Katy wrote this great piece on how to partner with Ushahidi that answers a lot of common questions: https://www.ushahidi.com/blog/2018/03/23/so-how-would-we-work-with-ushahidi

Design @Erioldoesdesign                        @justinscherer                         can you take a look at how you think we could create a hack way to get this onto our website?

**Comment by caharding** [comment]
@Erioldoesdesign @justinscherer I know this is a dangling issue from this cycle, but we'd love to close it if possible. Any hack solution?

**Comment by justinscherer** [comment]
We could just add a FAQ section like we have on https://www.ushahidi.com/plans and turn Katy's thing into the content.

**Comment by rowasc** [comment]
@jrtricafort is this something we are going to work on soon? Should it go into a cycle (6/7/any) ?

**Comment by jrtricafort** [comment]
@rowasc I'm going to add this to the pricing plans epic. Everything that is part of .com/plans should be addressed as part of the same scope and epic.

**Comment by jrtricafort** [comment]
@justinscherer Circling back on this issue-- I think I missed previous conversations. Where on dotcom were we proposing this "How to work with Ushahidi" FAQ to be added?

**Comment by Erioldoesdesign** [comment]
@jrtricafort Seems like to me we can incorporate some of the key useful language used in to the existing plan module. e.g. 
Mapper/Surveyor with a line underneath the title that says "Great for Grassroots orgs that operate on a hoe string budget" 
Responder could say: "Great for growing grassroots organisations or for those who've just landed funding!

etc etc.

Also the lower section of the page could be less of a 'how to do functions in Ushahidi FAQs' and more of a 'I'm a this type of org and these features are best for you' so We'd need to look in teh CMS to create these new FAQs and have them surface on the plans page (I think this is what Justin was getting at)
The only problem here is I don't see many folks scrolling that far down the page and certainly not reading big blocks of text.

I can draw up a proposal google doc on the copy that could change to incorporate what is useful from the blog post and a bit of styling. I don't think it needs a new page/new section.

Alternatively I'm pretty sure we have access to the CMS so Design could look at making these changes. @sethburtonhall how complex is changing/editing these modules in Craft?

**Comment by sethburtonhall** [comment]
@Erioldoesdesign Yes, please send that to me once you have it and I will integrate into the CMS. I will also update on where (possibly even if?) it can be updated in the CMS moving forward.

**Comment by sethburtonhall** [comment]
@rowasc I would vote yes, because it will be a quick revision.

**Comment by rowasc** [comment]
@jrtricafort adding this for Seth since he already noted that there's not enough work for him this cycle in a previous conversation.

**Comment by justinscherer** [comment]
where did we land on this?

**Comment by sethburtonhall** [comment]
I am waiting on @Erioldoesdesign per comment above.

> @Erioldoesdesign Yes, please send that to me once you have it and I will integrate into the CMS. I will also update on where (possibly even if?) it can be updated in the CMS moving forward.

**Comment by justinscherer** [comment]
Quick note on this: @caharding  @katymoney27 @jrtricafort Are we not going to change the pricing plans and rename them soon? Might make sense to do the change then.

**Comment by jrtricafort** [comment]
Thanks for circling back on this @justinscherer 

Question is for @katymoney27 : Do we still need this task to be completed, or will the upcoming Enterprise Page changes take care of this? (I'll ping you on this in Slack as well as requested)

@rowasc We can remove this from the current cycle until we hear back from Katy.

cc @sethburtonhall @Erioldoesdesign .

**Comment by Erioldoesdesign** [comment]
I actually did a doc incoporating some of the key elements on Katy's blog post:

https://docs.google.com/document/d/13CiZbbL_0WxEBeUSIUjo5sD5bQkBFFLOVPQj0PE8Xek/edit?usp=sharing

It includes 
- A new section for Int dev, humanitarian uni's, research org  and foundations, funders.
- Blue links in the top sections that anchor link to the FAQ question that answers that question or gives more details
- Text in the Mapper and Enterprise section at the top that says "Great for Grassroots organisations" and "Great for International Development, Humanitarian, University or Research Organizations" respectively
- 1 new FAQ question that is "How is Ushahidi funded?"

Formatting shouldn't change for the visuals. Use what is already there layout wise. What this does is try to connect up short statements in the 'table' with longer answers in FAQS that people may never see because they don't scroll.

@katymoney27 What do you think?

**Comment by jrtricafort** [comment]
@rowasc This is still in play. Adding it to the new epic: https://github.com/ushahidi/platform-cloud-interface/issues/318

**Comment by rowasc** [comment]
@jrtricafort A couple of questions
-  Is this the final content?  
-  what's the deadline or this? I think we should wait until ushahidi/platform-cloud-interface#318 is implemented and ready to launch to do this, right? 

cc @sethburtonhall

**Comment by rowasc** [comment]
@jrtricafort @Erioldoesdesign can one of you update the issue with the final content/ the link to the final content? 
Thanks!

**Comment by Erioldoesdesign** [comment]
Hmm @jrtricafort @rowasc Due to the changes like Mapper-> Demo etc. my document doesn't make much sense anymore...

It'll need meshing with page 15-17 this one: https://docs.google.com/document/d/11sRWll-Irpmb2r_Py19b6P7PO1EjMFj90sNFm7C8WLI/edit#

Will do that asap and then it'll need another review from @katymoney27

**Comment by rowasc** [comment]
Excellent thanks. Sent it to "In review" assigned to Katy when ready and I'll follow up too :D

**Comment by jrtricafort** [comment]
Thanks @Erioldoesdesign ! That sounds great. And yup, @katymoney27 should be the primary reviewer here. (Katy, let me know if you need additional eyes to review after Eriol provides updated version).

**Comment by Erioldoesdesign** [comment]
Updated my document: https://docs.google.com/document/d/13CiZbbL_0WxEBeUSIUjo5sD5bQkBFFLOVPQj0PE8Xek/edit?usp=sharing


This kind of ended up as duplicate work in a few ways but I saw value in doing it. @katymoney27 please review the document when you can because It may not meet the pricing work standards now. https://docs.google.com/document/d/13CiZbbL_0WxEBeUSIUjo5sD5bQkBFFLOVPQj0PE8Xek/edit?usp=sharing

The parts where the blog post info was worked in was the 'Specific industries' section though so that could be potentially pulled out of this work and done as a solo piece.

**Comment by rowasc** [comment]
@katymoney27 after you review and approve this doc from @Erioldoesdesign ,  please edit the description with the update link and assign it to @sethburtonhall  so he can implement it(or if you need to, just ping me and I'll do it)
Thanks

**Comment by sethburtonhall** [comment]
I am keeping an eye on this and will implement when we launch new pricing.

**Comment by Erioldoesdesign** [comment]
@rowasc It's been somewhat merged into other pricing work. I would say that this is 'in FE dev' right now. @sethburtonhall would you agree?

**Comment by rowasc** [comment]
Assigning to @sethburtonhall  to confirm the pipeline/status of this.See Eriol's message above

**Comment by sethburtonhall** [comment]
I believe everything is ready for testing in the staging environment.

http://darwin.ushahidi.com/pricing

**Comment by Erioldoesdesign** [comment]
@sethburtonhall Will do a second pass for Design QA and read all the copy in the morning 👍

**Comment by Erioldoesdesign** [comment]
So there were some comments in ticket #https://app.zenhub.com/workspace/o/ushahidi/platform-cloud-interface/issues/322

In the interests of clarity here they are again:
1 - Usertesting revealed last week that rows/columns 'matching' are important for a quick scan of features. The row title 'Catalogue of Services' with the yellow background would need to be all in one line. That would mean the 'Self Service' columns may have some blank space after 'No targeted surveys' but I believe that is more beneficial than the staggering of the column header 'Catalogue of Services'.

2 - The FAQ's need a bit more space between the answer and the next the bold header question (h3). They look a little squished together right now:
<img width="326" alt="screen shot 2018-09-03 at 10 27 09" src="https://user-images.githubusercontent.com/11681324/45090203-960aa400-b106-11e8-935b-864a6a968317.png">


I've checked over the copy and there are no errors. There are a couple of differences these are the things I'd like to be changed if possible:

- Remove the '*' before limited use
<img width="153" alt="screen shot 2018-09-05 at 12 17 32" src="https://user-images.githubusercontent.com/11681324/45090252-bc304400-b106-11e8-8be1-1340f152e23e.png">

- A paragraph break after '...cutomization' and 'Integrate...' so it looks like this:
<img width="251" alt="screen shot 2018-09-05 at 12 18 41" src="https://user-images.githubusercontent.com/11681324/45090288-d538f500-b106-11e8-9042-20014b564d11.png">

- There shouldn't be a dividing line between these two sections 'embeddable forms and roles...':
<img width="508" alt="screen shot 2018-09-05 at 12 20 54" src="https://user-images.githubusercontent.com/11681324/45090317-e8e45b80-b106-11e8-992e-8aedb08a8d4c.png">


I'd like to stress that because the two right enterprise columns have so much more content it's near impossible to scan read across rows to look at anything past price. What I'd like to *fix in a future improvement*:

- Have cell rows line up with cell rows. Users should be able to scan across rows of info. Let's not be afraid of 'empty' white space here.

- I'd like to remove the form fields in the enterprise columns and have a single CTA that takes the user to a full form. At the moment users fill out name, email, phone and org and then click the button to then fill out *more* info. This is not good UX.

- We have some duplicate cells. Pricing is stated twice in different cells and i suspect users find the various numbers detailed confusing. I would suggest we try something like "$15,000+$1,500+$999+/month$1500+/month"
Or Flat fee: "$16,500+"
Monthly costs: "$2499+/month"
Let's do as many calculations for users as possible.

Tagging @jrtricafort and @katymoney27 for transparency on my future hopes for this page :)

**Comment by sethburtonhall** [comment]
Thanks @Erioldoesdesign see revisions here http://darwin.ushahidi.com/pricing

**Comment by rowasc** [comment]
Closing as Seth released it yesterday
